plugin-server/src/config/config.ts

```diff
@@ -137,6 +137,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         RUSTY_HOOK_ROLLOUT_PERCENTAGE: 0,
         RUSTY_HOOK_URL: '',
         CAPTURE_CONFIG_REDIS_HOST: null,
+        HEATMAPS_PROCESSING_ENABLED: true,
```

Comment:
it's not really an experimental feature... 
couldn't this be a deny list of team ids or tokens?

Reply:
Could we indeed turn this PR into a team token list instead?

Reply:
It is still marked alpha on the app. It should source the team setting from the TeamManager like blobby does to drop server-side, but that's a more involved PR than I'm comfortable doing on a Friday afternoon. Happy to review such a PR.
<img width="243" alt="image" src="https://github.com/PostHog/posthog/assets/6241083/2405b755-4093-480d-8f6b-e15a19ed1597">

Reply:
Perfect! Closing as superseeded by https://github.com/PostHog/posthog/pull/23328

Reply:
Only the in app UI is alpha. The toolbar UI has been in "production" for a while now